WeatherStation/v2/_init.yaml

```diff
@@ -0,0 +1,9 @@
+# A configuration that may be suitable for testing.
+instances:
+  - sal_index: 1
+    ws_type: lsst
+    config:
+      host: localhost
+      port: 50000
+      buffer_size: 2048
+      timeout: 120.0
```

Comment:
You may, if you like, omit the values for port, buffer_size, and timeout if you specify defaults in the LSST weather station's config schema. For such a small schema it may be nice to provide values here. But if you end up with many weather stations or a lot more configuration, I would lean towards omitting defaulted values -- especially those that you are unlikely to ever change.

Reply:
Done and see my previous question.